content/en/docs/tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh

```diff
@@ -18,7 +18,7 @@
 source "tests/util/gateway-api.sh"
 install_gateway_api_crds
 
-# @setup profile=none
+# @setup profile=minimal
```

Comment:
hope this change is intended?

Reply:
It is. See #14091 for the reason it was made in `master` and `1.20`.